code/game/objects/structures/barricade/barricade.dm

```diff
@@ -419,20 +419,21 @@
 
 	var/obj/item/weapon/gun/smg/nailgun/nailgun = item
 
-	if(!nailgun.in_chamber || !nailgun.current_mag || nailgun.current_mag.current_rounds < 3)
-		to_chat(user, SPAN_WARNING("You require at least 4 nails to complete this task!"))
-		return FALSE
-
 	// Check if either hand has a metal stack by checking the weapon offhand
 	// Presume the material is a sheet until proven otherwise.
 	var/obj/item/stack/sheet/material = null
+	var/obj/item/weapon/gun/smg/nailgun/compact/tactical/mat_tac = 4 //Material needed to make a repair with a tactical compact nailgun.
```

Comment:
Bad typing. Just use var/mat_tac if this is a number. Also it looks like it's never changed so you might want to make it a define, or just use the number directly

Reply:
Yep meant to fix that. Thanks. It may change in the future so Im going to leave it as a var so the change only has to happen in one place.